conn.go

```diff
@@ -2089,7 +2089,11 @@ func (res *rows) Next(dest []driver.Value) (err error) {
 			cn.saveMessage(response, &responseBuf)
 			res.readTuples(dest)
 			return
-
+		case 'E':
+			cn.recv_n_bytes(4)
+			length, _ := cn.recv_n_bytes(4)
+			responseBuf, _ := cn.recv_n_bytes(int(length.int32()))
+			elog.Fatalf(chopPath(funName()), "%s", responseBuf.string())
```

Comment:
```suggestion
		case 'E':
		            cn.recv_n_bytes(4)
                            length, _ := cn.recv_n_bytes(4)
                            responseBuf, _ := cn.recv_n_bytes(int(length.int32()))
                            errorString := responseBuf.string()
                           err = errors.New(errorString)
                           elog.Infoln(funName(), errorString)
                           return err
                case 'Z': /* Backend is ready for new query (6.4) */
                           return err

```
elog.Fatalf() will cause the application to crash hence a better approach is to log error and continue with further processing.
I have also added code to handle 'Z' from backend. This was missing in original code.